frontends/web/src/routes/account/info/signingconfiguration.tsx

```diff
@@ -79,19 +101,20 @@ class SigningConfiguration extends Component<Props, State> {
                     </div>
                 </div>
                     :
-                info.xpubs.map((xpub, index) => {
+                info.xpubs.map((xpub, xpubIndex) => {
                     return (
                         <div key={xpub}>
+                            <h2>{this.scriptTypeTitle(info.scriptType)}</h2>
                             <label className="labelLarge">{t('accountInfo.extendedPublicKey')}</label>
```

Comment:
This subtitle, "Extended public key" is now repeated for every scripttype / derivation path, 
how about moving it into the Header next to or below "Account Information" and also add  xpub 
i.e. "Extended public key (xpub)"
WDYT?

Reply:
ah just realized that wouldn't work for legacy, then it just says accountInfo.address
```
  "accountInfo": {
    "address": "Address",
    "extendedPublicKey": "Extended public key",
```

Reply:
@thisconnect if you don't mind, i prefer to make the MVP in the frontend, and then pass it on to you to make it nice :innocent: ok?

Reply:
legacy also has an xpub. watch-only only have an address.

Reply:
👍